app/main/updater.js

```diff
@@ -0,0 +1,56 @@
+import {dialog} from 'electron';
+import {autoUpdater} from 'electron-updater';
+
+import {t} from './helpers';
+
+const RELEASES_LINK = 'https://github.com/appium/appium-inspector/releases';
+
+autoUpdater.autoDownload = false;
+autoUpdater.autoInstallOnAppQuit = false;
+
+autoUpdater.on('error', (error) => {
+  dialog.showErrorBox(t('Could not download update'), t('updateDownloadFailed', {message: error}));
+});
+
+autoUpdater.on('update-not-available', () => {
+  dialog.showMessageBox({
+    type: 'info',
+    buttons: [t('OK')],
+    message: t('No update available'),
+    detail: t('Appium Inspector is up-to-date'),
+  });
+});
+
+autoUpdater.on('update-available', async ({version, releaseDate}) => {
+  const pubDate = new Date(releaseDate).toDateString();
+  const {response} = await dialog.showMessageBox({
+    type: 'info',
+    message: t('appiumIsAvailable', {name: version}),
+    buttons: [t('Install Now'), t('Install Later')],
+    detail: t('updateDetails', {pubDate, notes: RELEASES_LINK}),
```

Comment:
would it be possible to show release notes for the particular version?

Reply:
I did investigate this option, but decided against it for several reasons:
* The raw release notes string requires post-processing to make it readable (remove HTML tags with regex, handle multi-level bullet points, etc)
* The release notes can get quite large and cause the floating box buttons to go out of bounds, preventing any user interaction. I mainly observed this with the raw release notes string, but there is still a possibility that it can apply to the post-processed notes in the future. A quick search for scrolling options inside this box also didn't give me any results.
* The user may skip one or more versions during the update, and showing the changelog only for the most recent version is not sufficient, in my opinion

Still, the floating box does have another problem - it cannot render clickable links, so the link shown here is unfortunately just text.